### DISCUSSION TITLE
Login

### DISCUSSION THREAD

**Discussion by sajkofff** [rejetto/hfs#515] [Q&A] [answered]
Hello. Is it possible to make sure that when you click on a free area on the login screen, the password entry window does not disappear?

Additional question. Can I change the "username" and "password" labels to something else?

![2024-03-10_004002](https://github.com/rejetto/hfs/assets/48749271/148cb80c-c2d7-44e9-8e94-23145d677050)

**Comment by rejetto** [comment]
i took the chance to add this thing to the documentation
https://github.com/rejetto/hfs/wiki/Translation#customization

**Comment by sajkofff** [comment]
Grazie mille. 
Didn't know about this.  How about my first question.  If you click on a free area, the password entry window disappears.  You have to refresh the page.  However, in my organization, not everyone can guess that the page needs to be updated.

**Comment by rejetto** [comment]
that cannot tbe disabled.
how did the dialog open in the first place? did the user click" login" ?

**Reply by sajkofff** [reply]
Login to the resource is restricted for users, so a login dialog appears.  Everything is fine with this.  But if you accidentally click somewhere else, the login form disappears.

I mean if you click not on the login form, but on the blurred background.

**Reply by sajkofff** [reply]
![Video_2024-03-10_104031](https://github.com/rejetto/hfs/assets/48749271/7d2570a3-bda7-415c-947f-e4989203bd36)

**Comment by rejetto** [comment]
i see, that doesn't seem ideal

**Comment by rejetto** [comment, marked as answer]
clicking the text on top will bring the login dialog back, but that's no solution of course.
I guess closing the dialog in such case makes no sense, so it's best to prevent it.
This should solve [52-beta5.2](https://github.com/rejetto/hfs/files/14550410/hfs-windows.zip)
To install it quickly you can right-click on the "check for updates" button

**Reply by sajkofff** [reply]
Unfortunately it doesn't work

**Reply by rejetto** [reply]
what's the problem

**Reply by sajkofff** [reply]
Sorry. Apparently I did something wrong. I repeated the update procedure. Everything is working. Thank you
